NOTICE.txt

```diff
@@ -0,0 +1,33 @@
+azure-sdk-for-c
+
+NOTICES AND INFORMATION
+Do Not Translate or Localize
+
+This software incorporates material from third parties. Microsoft makes certain
+open source code available at https://3rdpartysource.microsoft.com, or you may
+send a check or money order for US $5.00, including the product name, the open
```

Comment:
What do I get if I send $5 to this address?

Reply:
That was unclear to me as well.  Its verbatim text from MSFTs 4rd party source code site.  Perhaps they send you a usb drive with the cotents of the download?

FYI, this is the details from the site when you go to "compliance inquiry".

License and compliance inquiry
Microsoft products often incorporate material from third parties. Microsoft makes certain open source code available at https://3rdpartysource.microsoft.com, or you may send a check or money order for US $5.00, including the product name, the opens source component name, architecture, and version number, to:

Source Code Compliance Team
Microsoft Corporation
One Microsoft Way
Redmond, WA 98052
USA

Please contact opensource@microsoft.com with any questions or concerns.

Reply:
Although they don't say it on the site, the team confirmed to me.  They will burn a CD with the contents of the "[download]" link.

Reply:
Following up offline